api/src/main/java/org/pmiops/workbench/cohorts/CohortMaterializationService.java

```diff
@@ -223,6 +252,50 @@ private void addFilterOnConcepts(TableQuery tableQuery, Set<Long> conceptIds,
     }
   }
 
+  @VisibleForTesting
+  CdrQuery getCdrQuery(@Nullable CohortReview cohortReview, SearchRequest searchRequest,
```

Comment:
opt_nit: in general (barring inconsistency with other related methods) I find sorting parameters in descending order of importance to be the most readable, so usually I'd expect to find nullables towards the end.

Reply:
Will follow up on this.